src/pages/guides/how-google-fixes-vulnerabilities.tsx

```diff
@@ -0,0 +1,48 @@
+import { FunctionComponent } from 'react'
+
+import { Layout, GatedResourceLayout } from '@components'
+
+export const Guide: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'How Google and Others and Fix Vulnerabilities | Sourcegraph',
+            description:
+                'See how tech companies large and small address code security. In this post, we explore how Google, Microsoft, Lyft, GitLab, and Atlassian find and fix vulnerabilities.',
+        }}
+        className="navbar-white"
+    >
+        <GatedResourceLayout
+            title="How Google, Microsoft, Lyft, GitLab, and Atlassian find and fix vulnerabilities"
+            formLabel="Download the e-book"
+            form={{
+                formId: '610debf1-bbce-45d6-a492-6858b5b3db18',
+                onFormSubmitted: () => window.open('/guides/sg-how-companies-fix-vulnerabilities.pdf'),
```

Comment:
As an improvement, we could name the PDF with the title of the PDF instead of it showing the filename in the tab. Check out [this comment for reference](https://github.com/sourcegraph/about/pull/5498#pullrequestreview-1015357937). 🤓

Reply:
What a helpful tool. Thanks for the rec, Brett! I've updated the PDF here, which displays the title now in browser. I will update the file in the paired ticket as well.

<img width="475" alt="Screen Shot 2022-06-23 at 11 05 27 AM" src="https://user-images.githubusercontent.com/60713139/175344489-737599a1-92fa-4232-809e-bb2c785de35e.png">

Reply:
Awesome!! Thanks @zlonko!